test/unit/ButtonCheckbox.spec.js

```diff
@@ -0,0 +1,98 @@
+/* global describe, it, expect, beforeEach */
+
+import Vue from 'vue/dist/vue.js'
+import VkButton from '../../src/lib/Button.vue'
+import VkButtonCheckbox from '../../src/lib/ButtonCheckbox.vue'
+
+describe('ButtonCheckbox.vue', () => {
+  var vm
```

Comment:
cannot make it as a `const` here, as need to define it globally for all tests, and init `vm` in `beforeEach()`